src/constants/notificationSchema.js

```diff
@@ -0,0 +1,71 @@
+export const notificationSchema = {
+  collaboration_manager_create: {
```

Comment:
maybe these property names should come from `notificationTypeNames` above? seems like they would need to be kept in sync manually this way